examples/bof_detection/compile.sh

```diff
@@ -0,0 +1 @@
+gcc bof.c -o bof -fno-stack-protector -std=c99
```

Comment:
Maybe just put this on top of the bof.c file?
`//gcc bof.c -o bof -fno-stack-protector -std=c99`